docs-src/modules/developing/pages/materializations.adoc

```diff
@@ -157,5 +157,10 @@ session = LocalBuilder().build()
 materialization.delete_materialization("PurchaseStats")
 ----
 
+[source,bash]
```

Comment:
should this be `[source,cli]` or `[source,kaskada-cli]` to get the mouseover client name to be "more" correct?

Reply:
The second argument specifies the syntax highlighter. We could provide a highlighter for the CLI, but I think it would be better to just use the Bash highlighting.